utils/build_appimage.sh

```diff
@@ -17,7 +17,15 @@ mv AppDir/usr/share/games/endless-sky/* AppDir/
 
 # Now build the actual AppImage
 curl -sSL https://github.com/linuxdeploy/linuxdeploy/releases/download/continuous/linuxdeploy-x86_64.AppImage -o linuxdeploy && chmod +x linuxdeploy
-./linuxdeploy --appdir AppDir -e endless-sky -d endless-sky.desktop -i endless-sky.png --output appimage
+OUTPUT=Endless_Sky-continuous-x86_64.AppImage ./linuxdeploy --appdir AppDir -e endless-sky -d endless-sky.desktop -i endless-sky.png --output appimage
+
+# Use the static runtime for the AppImage. This lets the AppImage being run on systems without fuse2.
+APPTOOL=$(curl -sSL https://github.com/probonopd/go-appimage/releases/expanded_assets/continuous | grep "appimagetool-.*-x86_64.AppImage" | head -n 1 | cut -d '"' -f 2)
```

Comment:
It's probably better to grep our href specifically instead of relying on it being the first in the list:

```suggestion
APPTOOL=$(curl -sSL https://github.com/probonopd/go-appimage/releases/expanded_assets/continuous | grep -Eo 'href="(/.*?appimagetool-.*-x86_64.AppImage)"' | cut -d '"' -f 2)
```

Also, if we make one further assumption (`gh` being installed), we can do a much less hacky
```
APPTOOL=$(gh release download -R probonopd/go-appimage continuous -p appimagetool*x86_64.AppImage)
```